package.json

```diff
@@ -12,11 +12,11 @@
     "generate:middleware-types": "graphql-codegen -r dotenv/config",
     "fetch-definitions": "cross-env node scripts/fetchDefinitions.js",
     "update-readme": "cross-env node scripts/updateReadme.js",
-    "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --package polymesh-types --input ./src/polkadot",
-    "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package polymesh-types --endpoint ws://localhost:9944 --output ./src/polkadot --strict",
+    "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --endpoint metadata.json --package polymesh-types --input ./src/polkadot/",
+    "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --endpoint metadata.json --package polymesh-types --output ./src/polkadot --strict",
```

Comment:
It seemed like one of these commands didn't like the ws endpoint with the chain.

I uploaded a metatdata.json to use to drive: https://drive.google.com/file/d/1137jNjQZYdSNOkXtk42LYv8yD-kOkcHP/view?usp=sharing

I used: `curl -H "Content-Type: application/json" -d '{"id":"1", "jsonrpc":"2.0", "method": "state_getMetadata", "params":[]}' http://localhost:9933 > metadata.json` to get it.

Reply:
What was the specific problem with using the endpoint?

Reply:
It kept on thinking it was a file

`Error: Cannot find module '/Users/eric/polymesh-sdk/ws:/localhost:9944'`

Reply:
Thats with:

```
  "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --endpoint  metadata.json --package polymesh-types --input ./src/polkadot/",
    ```
    
    I had to check again since the error stripped a / it seems. the meta one seems fine though

Reply:
Seems like they fixed it in a May 8th release. The note says `Deupe wss:// handling in polkadot-types-from-{chain, defs}`. I tried it out and web socket seems to work.

However, it seems to change some more types and has more breaking changes in it for us, even though its a minor version. I wan to get this merged in before making more changes.

Reply:
Created a task to track this